apps/src/templates/instructions/CommitsAndReviewTab.jsx

```diff
@@ -115,8 +127,15 @@ const CommitsAndReviewTab = props => {
           />
         </div>
       </div>
-      <CodeReviewTimeline reviewData={reviewData} commitsData={commitsData} />
-      {!hasOpenCodeReview && (
+      <CodeReviewTimeline
+        timelineData={[
+          ...timelineData,
+          ...(openReviewData ? [openReviewData] : [])
```

Comment:
(minor) I think you might not need the ternary here.  Does `...` on `null` or `undefined` give the behavior you want?

Reply:
<img width="346" alt="Screen Shot 2022-05-10 at 10 28 55 AM" src="https://user-images.githubusercontent.com/24235215/167687622-f8189047-10f7-48a2-9905-ae636c6df1fe.png">